zetasql/parser/bison_parser.y

```diff
@@ -2197,6 +2222,32 @@ function_parameters:
       }
     ;
 
+user_list:
+    user_info
+      {
+        $$ = MAKE_NODE(ASTUserList, @$, {$1});
+      }
+    | user_list "," user_info
+      {
+        $$ = WithEndLocation(WithExtraChildren($1, {$3}), @$);
+      }
+    ;
+
+user_info:
+    identifier_or_string opt_password
```

Comment:
we can make options followed by **identifier** more extensible. `IDENTIFIED BY` is good but I guess **options_list** better:
```yacc
CREATE USER name OPTIONS ( password = ..., others = ... )
```
And `PASSWORD` do not have to be keyword.

Reply:
it's better to use the existing syntax like mysql/tidb.

Reply:
it's not extensible, CREATE USER if not SQL standard, syntax is left to implementation.

Reply:
we cannot add option at the end of  the list? if there are other concerns, we can limit creating one user at most

Reply:
pls check 'create table' 's usage of options. Our primary target SQl dialect is, more likely SparkSQL, there is no need to follow a non-standard implementation from MySQL. 

'Create User' should introduce more features (at least I think), with options_list u don't have to update zetasql each time.

Reply:
the user experience and habits are the primary consideration. i do not think adding non standard syntax at first is a good decision

Reply:
MySQL is not standard